extension/src/dashboard/pages/Bookmarks/index.tsx

```diff
@@ -34,7 +34,7 @@ const Bookmarks = () => {
 
       <Switch>
         <Route path="/favoris/souscriptions" component={SubscriptionsTab} />
-        <Route exact path="/favoris" component={BookmarksTab} />
+        <Route path="/favoris" component={BookmarksTab} />
```

Comment:
Why don't we keep `exact`?

Reply:
When `exact`, the route `/favoris/:id` doesn't match so the `BookmarksListWithInformation` won't display